src/strings/en-us.json

```
@@ -119,7 +119,7 @@
     "ButtonPause": "Pause",
     "ButtonPlayer": "Player",
     "ButtonPreviousTrack": "Previous track",
-    "ButtonQuickStartGuide": "Quick Start Guide",
+    "ButtonQuickStartGuide": "Post-Install Setup Guide",
```

I think calling it "Post-Install" while you are going through the first time setup process to be confusing...

```suggestion
    "ButtonQuickStartGuide": "Setup Guide",
```